src/style.css

```diff
@@ -1,3 +1,80 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+
+  --header-height: 60px;
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  min-width: 1024px;
```

Comment:
The `min-width: 1024px;` on the `body` element may cause layout issues on smaller screens or devices. Consider using responsive design techniques, such as media queries, to ensure the layout adapts to different screen sizes.